### ISSUE TITLE
breakable blood vessels or breakable springs

### ISSUE THREAD

**Issue by cuptaza** [studio-minus/ppg-bugs-and-requests#2452]
### Feature description

add a version of the blood vessel/spring that is breakable

### Feature purpose

because, there are many cables that got a breakable version (like the cable 'n' stuff)
i asked for this mostly because, many scenes could be made whit this function where the vessels are supposed to break.

**Comment by ATPGaming** [comment]
Oh god imagine a destructible model of a human with veins that you can break

**Comment by sp22222222** [comment]
i agree with this post as made evident by my most recent one on this forum #2464

**Comment by DindinYT37** [comment]
> i agree with this post as made evident by my most recent one on this forum #2464

This is an issue-board, not a forum. Also, please stop spamming comments by commenting "I agree with this post" on everything – we have a thumbs up reaction for that.

**Comment by cuptaza** [comment]
> Oh god imagine a destructible model of a human with veins that you can break
> 
> I'm not sure about the destructible spring though, it seems if it broke from stretching considering springs are made to do that
the destructible spring should be like, if you strech it TOO much, like; pulling the spring from a corner of a map to another corner,atacching a slider and then freezing one of them and use the push power TOO MUCH
something like that
